src/table/cells/__tests__/handle-cell-focus.spec.js

```diff
@@ -0,0 +1,67 @@
+import * as handleCellFocus from '../handle-cell-focus';
+
+describe('handle-key-press', () => {
+  describe('focusCell', () => {
+    const rowElements = [
+      {
+        getElementsByClassName: () => [{ focus: sinon.spy(), setAttribute: sinon.spy() }],
+      },
+    ];
+    const nextCellCoord = [0, 0];
+    const nextCell = handleCellFocus.focusCell(rowElements, nextCellCoord);
+    expect(nextCell.focus).to.have.been.calledOnce;
+    expect(nextCell.setAttribute).have.been.calledOnce;
+  });
+
+  describe('handleBodyCellFocus', () => {
+    const sandbox = sinon.createSandbox();
+    const cell = {
+      rawRowIdx: -1,
+      rawColIdx: 0,
+    };
+    const focusedCellCoord = {
+      current: [0, 0],
+    };
+    const rootElement = {
+      getElementsByClassName: () => [
+        { getElementsByClassName: () => [{ focus: sinon.spy(), setAttribute: sinon.spy() }] },
```

Comment:
```suggestion
    const cell = { focus: sinon.spy(), setAttribute: sinon.spy() };
    const rootElement = {
      getElementsByClassName: () => [
        { getElementsByClassName: () => [cell] },
```
If you do something like this I don't think you have to return the cell. Shouldn't have returns unless it is really needed

Reply:
Perfect. Thanks!